src/server/node_manager.c

```diff
@@ -1949,9 +1950,9 @@ find_degraded_occurrence(resc_resv *presv, struct pbsnode *np,
 		if (find_vnode_in_execvnode(execvnodes_seq[i], np->nd_name)) {
 			occr_found = 1;
 			if (degraded_op == Set_Degraded_Time) {
-				/* we keep track of the occurrence time to determine the earliest
-				 * degraded time */
-				occr_time = get_occurrence(rrule, dtstart, tz, j);
+		/* we keep track of the occurrence time to determine the earliest
```

Comment:
Please indent this comment and the line below.

Reply:
Done